chapter_crashcourse/ndarray.md

```diff
@@ -132,23 +133,24 @@ For stylistic convenience, we can write `y.exp()`, `x.sum()`, `x.norm()`, etc. a
 
 ## Broadcast Mechanism
 
-In the above section, we saw how to perform operations on two NDArrays of the same shape. When their shapes differ, a broadcasting mechanism may be triggered analogous to NumPy: first, copy the elements appropriately so that the two NDArrays have the same shape, and then carry out operations by element.
+In the above section, we saw how to perform operations on two ndarrays of the same shape. When their shapes differ, a broadcasting mechanism may be triggered: first, copy the elements appropriately so that the two ndarrays have the same shape, and then carry out operations by element.
 
 ```{.python .input  n=14}
 a = np.arange(3).reshape((3, 1))
 b = np.arange(2).reshape((1, 2))
-a, b
+print('a : ', a)
+print('b : ', b)
 ```
 
-Since `a` and `b` are (3x1) and (1x2) matrices respectively, their shapes do not match up if we want to add them. NDArray addresses this by 'broadcasting' the entries of both matrices into a larger (3x2) matrix as follows: for matrix `a` it replicates the columns, for matrix `b` it replicates the rows before adding up both element-wise.
+Since `a` and `b` are (3x1) and (1x2) matrices respectively, their shapes do not match up if we want to add them. We 'broadcast' the entries of both matrices into a larger (3x2) matrix as follows: for matrix `a` it replicates the columns, for matrix `b` it replicates the rows before adding up both element-wise.
 
 ```{.python .input}
 a + b
 ```
 
 ## Indexing and Slicing
```

Comment:
At the beginning, show example of `x[1]` and highlight that its result is a scalar.